orange-stock-ios/orange-stock-ios/View/FavoriteStockListViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  FavoriteStockListViewController.swift
+//  orange-stock-ios
+//
+//  Created by hogang on 2023/11/20.
+//
+
+import UIKit
+import SnapKit
+
+// View: 나의 관심 주식 목록
+
+class FavoriteStockListViewController: UIViewController {
```

Comment:
```suggestion
final class FavoriteStockListViewController: UIViewController {
```
더이상 상속될 가능성이 없는 클래스는 항상 `final` 키워드로 선언해주길 권장드립니다.